src/resources/container.js

```diff
@@ -60,6 +61,62 @@ class ContainerResource {
     instantiateRenderEntity(options) {
         return null;
     }
+
+    /**
+     * Queries the list of available material variants.
+     *
+     * @returns {string[]} An array of variant names.
+     * @example
+     * // load a glb file and instantiate an entity with a render component based on it
+     * app.assets.loadFromUrl("statue.glb", "container", function (err, asset) {
+     *     var entity = asset.resource.instantiateRenderEntity({
+     *         castShadows: true
+     *     });
+     *     app.root.addChild(entity);
+     *     var materialVariants = asset.resource.getMaterialVariants();
+     *     var variantListOptions = materialVariants.map((variant) => ({ v: variant, t: variant }));
+     */
```

Comment:
I'm not sure this example is needed, as it's essentially the same as for the other function.